modules/books.js

```diff
@@ -0,0 +1,64 @@
+const bookList = document.querySelector('.book-list');
+
+export default class Books {
+  constructor(bookLists = []) {
+    this.bookLists = bookLists;
+    this.getFromLocal();
+  }
+
+  removeBook = (bookObject, index) => {
+    const bookInfo2 = document.getElementById(index);
+    const { title, author } = bookObject;
+
+    this.bookLists = this.bookLists.filter(
+      (book) => book.title !== title && book.author !== author,
+    );
+    localStorage.setItem('booksCollection', JSON.stringify(this.bookLists));
+    bookList.removeChild(bookInfo2);
+  };
+
+  displayBook = (bookObject, index) => {
+    const bookInfo = document.createElement('div');
+    bookInfo.classList = 'bookInfo';
+    bookInfo.id = index;
+
+    bookInfo.innerHTML = `
+      <p class='book-details'>'${bookObject.title}' by ${bookObject.author}</p>
+    `;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.classList = 'remove-btn';
+    removeBtn.innerText = 'Remove';
+
+    bookInfo.appendChild(removeBtn);
+    bookList.prepend(bookInfo);
+
+    removeBtn.onclick = () => {
+      const conf = window.confirm('Are you sure you want to remove this book?');
```

Comment:
- [ ] Please don't use **window.confirm** in your code because it behaves differently on different browsers and sometimes it stops the application from working. :cry: If you wanna show errors then use custom modals according to [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#javascript-in-the-browser) :+1: